### PR TITLE
add r-platetools

### PR DESCRIPTION
**Pull request by gwaygenomics**
`@conda-forge/help-r`

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

I used @bgruening's [`conda_r_skeleton_helper`](https://github.com/bgruening/conda_r_skeleton_helper) to generate the recipe. I also included the custom LICENSE file in the `platetools` R package: https://github.com/Swarchal/platetools

(a conda-forge version of bioconda/bioconda-recipes#15189)

### CODE REVIEW

**Comment by gwaygenomics** [comment]
`appveyor` error:

> Preparing transaction: ...working... done
Verifying transaction: ...working... ERROR:conda.stderr:
SafetyError: The package for r-base located at C:\Miniconda36-x64\pkgs\r-base-3.5.1-hfa6e2cd_1006
appears to be corrupted. The path 'lib/R/doc/html/packages.html'
has an incorrect size.
  reported size: 6062 bytes
  actual size: 9437 bytes
done
Executing transaction: ...working... done
WARNING: symlink_conda() is deprecated.
Found sh test file on windows.  Ignoring this for now (PRs welcome)
WARNING:conda_build.build:Found sh test file on windows.  Ignoring this for now (PRs welcome)
set PREFIX=C:\bld\r-platetools_1557336874395\_test_env
set SRC_DIR=C:\bld\r-platetools_1557336874395\test_tmp
(base) %SRC_DIR%>call "%SRC_DIR%\conda_test_env_vars.bat" 
(base) %SRC_DIR%>call "C:\Miniconda36-x64\Scripts\activate.bat" "%PREFIX%" 
(%PREFIX%) %SRC_DIR%>IF 0 NEQ 0 exit 1 
(%PREFIX%) %SRC_DIR%>IF 0 NEQ 0 exit 1 
(%PREFIX%) %SRC_DIR%>call "%SRC_DIR%\run_test.bat" 
(%PREFIX%) %SRC_DIR%>"%PREFIX%\bin\R" -e "library('platetools')" 
The system cannot find the path specified.
(%PREFIX%) %SRC_DIR%>IF 3 NEQ 0 exit /B 1 
(%PREFIX%) %SRC_DIR%>IF 1 NEQ 0 exit 1 
Tests failed for r-platetools-0.1.2-r351h6115d3f_0.tar.bz2 - moving package to C:\bld\broken
WARNING:conda_build.build:Tests failed for r-platetools-0.1.2-r351h6115d3f_0.tar.bz2 - moving package to C:\bld\broken
TESTS FAILED: r-platetools-0.1.2-r351h6115d3f_0.tar.bz2
Command exited with code 1

**Comment by gwaygenomics** [comment]
perhaps this is ok? (https://github.com/conda-forge/staged-recipes/pull/8221#issuecomment-487365749)

**Comment by gwaygenomics** [comment]
@daler @bgruening - Looks like `r-platetools` built with `conda_r_skeleton_helper` passed osx and linux tests (win failures). Good to merge? Thanks 😄

File: recipes/r-platetools/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+YEAR: 2016
+COPYRIGHT HOLDER: Scott Warchal
```
Comment (bgruening) [inline recipes/r-platetools/LICENSE.txt]: That file looks strange. Its not the MIT license thats for sure :)

**Comment by bgruening** [comment]
Besides the license, +1 from me!

File: recipes/r-platetools/LICENSE.txt (hunk 1 above)
Comment (gwaygenomics) [reply recipes/r-platetools/LICENSE.txt]: yeah, good point. Maybe @Swarchal can clarify?

Comment (Swarchal) [reply recipes/r-platetools/LICENSE.txt]: It's unfortunately how CRAN want things formatted for submission. I can add an additional LICENSE.md file containing the full MIT license to the repo if that would fix things?

Comment (gwaygenomics) [reply recipes/r-platetools/LICENSE.txt]: In the meantime, I added the standard MIT license in 57db5c1

Comment (gwaygenomics) [reply recipes/r-platetools/LICENSE.txt]: Thanks @Swarchal! I must have written my previous response at the same time. Does the change in 57db5c1 look ok?

Comment (Swarchal) [reply recipes/r-platetools/LICENSE.txt]: If it doesn't need to pass CRAN checks then yes that's fine with me!

**Comment by gwaygenomics** [comment]
Anything else to do with this PR @bgruening ? Would love to get this merged in today 😃

**Comment by gwaygenomics** [comment]
Any updates on getting this package merged in? It appears these are expected windows errors.

Tagging some maintainers who have recently merged 😄@ocefpaf @dbast

**Comment by dbast** [comment]
Please also add `- conda-forge/r` to recipe-maintainers.

**Comment by gwaygenomics** [comment]
Thanks @dbast - done!

**Comment by dbast** [comment]
LGTM

**Comment by ocefpaf** [comment]
> Tagging some maintainers who have recently merged :smile:@ocefpaf @dbast

To review and merge staged-recipes you need to ping the @conda-forge/staged-recipes team. See https://github.com/conda-forge/staged-recipes#12-my-pull-request-passes-all-checks-but-hasnt-received-any-attention-how-do-i-call-attention-to-my-pr--what-is-the-customary-amount-of-time-to-wait

**Comment by ocefpaf** [comment]
Looks like @bgruening is more familiar with this package so I'll defer to him to merge.

**Comment by gwaygenomics** [comment]
> To review and merge staged-recipes you need to ping the @conda-forge/staged-recipes team. 

I would have but first time contributors cannot tag @conda-forge/staged-recipes 🙃 (I thought I saw this referred to in documentation somewhere, but I can't seem to find it now)

**Pull request merged**